editor/src/core/shared/runtime-report-logs.tsx

```diff
@@ -98,68 +89,6 @@ export function setReactRouterErrorHasBeenLogged(value: boolean): void {
   reactRouterErrorLogged = value
 }
 
-const ReactRouterErrorPrefix = `React Router caught the following error during render`
-const ReactRouterAwaitErrorPrefix = `<Await> caught the following error during render`
-
-export function useUpdateOnConsoleLogs(
-  referentiallyStableCallback: (newConsoleLogs: Array<ConsoleLog>) => void,
-): void {
-  useSubscribeToPubSubAtom(consoleLogsAtom, referentiallyStableCallback)
-}
-
-export function useReadOnlyConsoleLogs(): Array<ConsoleLog> {
-  return usePubSubAtomReadOnly(consoleLogsAtom, AlwaysTrue)
-}
-
-export function useWriteOnlyConsoleLogs(): {
-  addToConsoleLogs: (log: ConsoleLog) => void
-  clearConsoleLogs: () => void
-} {
-  const modifyLogs = usePubSubAtomWriteOnly(consoleLogsAtom)
-
-  const clearConsoleLogs = React.useCallback(() => {
-    modifyLogs(EmptyConsoleLogs)
-  }, [modifyLogs])
-
-  const addToConsoleLogs = React.useCallback(
-    (log: ConsoleLog) => {
-      modifyLogs((logs) => {
-        let result = [...logs, log]
-        while (result.length > ConsoleLogSizeLimit) {
-          result.shift()
-        }
-        return result
-      })
-      // For an error...
-      if (log.method === 'error') {
-        // ...where the first value...
-        const firstLine = log.data[0]
-        // ...is a string...
-        if (typeof firstLine === 'string') {
-          // ...and it starts with these prefixes...
-          if (
-            firstLine.startsWith(ReactRouterErrorPrefix) ||
-            firstLine.startsWith(ReactRouterAwaitErrorPrefix)
-          ) {
-            // ...Mark these as having been seen.
-            setReactRouterErrorHasBeenLogged(true)
```

Comment:
In PR #5084 this was added so that if a change caused an exception in something like an `<Await>` component, that a further change would result in the error being cleared. Does that now work without this logic?